doc/Command-Reference.md

```diff
@@ -6564,6 +6568,312 @@ This command is used to set the counter polling interval. Default is 20 seconds.
+  ```
+
+**config snmp contact add/del/modify**
+
+This command is used to add, delete, or modify the SNMP contact.
+
+- Usage:
+  ```
+  config snmp contact add <contact> <contact_email>
+  ```
+
+- Example (config snmp contact add <contact> <contact_email>):
+  ```
+  admin@sonic:~$ sudo config snmp contact add joe joe@contoso.com
+  Contact name joe and contact email joe@contoso.com have been added to configuration
+  Restarting SNMP service...
+  ```
+
+- Usage:
+  ```
+  config snmp contact del <contact>
+  ```
+
+- Example (config snmp contact del <contact>):
+  ```
+  admin@sonic:~$ sudo config snmp contact del joe
+  SNMP contact joe removed from configuration
+  Restarting SNMP service...
+  ```
+
+- Usage:
+  ```
+  config snmp contact modify <contact> <contact_email>
+  ```
+
+- Example (config snmp contact modify <contact> <contact_email>):
+  ```
+  admin@sonic:~$ sudo config snmp contact modify test test@contoso.com
+  SNMP contact test and contact email test@contoso.com updated
+  Restarting SNMP service...
+  ```
+
+**config snmp community add/del/replace**
+
+This command is used to add, delete, or replace the SNMP community.
+
+- Usage:
+  ```
+  config snmp community add <community> (RO | RW)
+  ```
+
+- Example (config snmp community add <community> (RO | RW))
+  ```
+  admin@sonic:~$ sudo config snmp community add testcomm ro
+  SNMP community testcomm added to configuration
+  Restarting SNMP service...
+  ```
+
+- Usage:
+  ```
+  config snmp community del <community>
+  ```
+
+- Example (config snmp community del <community>)
+  ```
+  admin@sonic:~$ sudo config snmp community del testcomm 
+  SNMP community testcomm removed from configuration
+  Restarting SNMP service...
+  ```
+
+- Usage:
+  ```
+  config snmp community replace <community> <new_community>
+  ```
+
+- Example (config snmp community replace <community> <new_community>):
+  ```
+  admin@sonic:~$ sudo config snmp community replace testcomm newtestcomm
+  SNMP community newtestcomm added to configuration
+  SNMP community newtestcomm replace community testcomm
+  Restarting SNMP service...
+  ```
+
+**config snmp user add/del**
+
+This command is used to add or delete the SNMP user for SNMPv3.
+
+- Usage:
+  ```
+  config snmp user add <user> (noAuthNoPriv | AuthNoPriv | Priv) (RO | RW) [[(MD5 | SHA | MMAC-SHA-2) <auth_password>] [(DES |AES) <encrypt_password>]]
+  ```
+
+- Example (config snmp user add testuser1 noauthnopriv ro):
+  ```
+  admin@sonic:~$ sudo config snmp user add testuser1 noauthnopriv ro
+  SNMP user testuser1 added to configuration
+  Restarting SNMP service...
+  ```
+
+- Example (config snmp user add testuser2 authnopriv ro sha testuser2_auth_pass):
+  ```
+  admin@sonic:~$ sudo config snmp user add testuser2 authnopriv ro sha testuser2_auth_pass
+  SNMP user testuser2 added to configuration
+  Restarting SNMP service...
+  ```
+
+- Example (config snmp user add testuser3 priv rw md5 testuser3_auth_pass aes testuser3_encrypt_pass):
```

Comment:
Simplify to `- Example:`. The command in parenthesis is displayed in the prompt below.